noty-android/app/composeapp/src/main/java/dev/shreyaspatil/noty/composeapp/component/action/NotyActions.kt

```diff
@@ -31,6 +31,19 @@ import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.unit.dp
 import dev.shreyaspatil.noty.R
 
+@Composable
+fun PinAction(onClick: () -> Unit, isPinned: Boolean) {
```

Comment:
This is workable. But it would be nice if you add `isPinned` as a first parameter and `onClick` as a second parameter.

Reply:
Like this:

```suggestion
fun PinAction(isPinned: Boolean, onClick: () -> Unit) {
```

Reply:
Sure, any specific reason we do that?

Reply:
Since the lambda parameter is not at the end of the parameter list, when anyone calls this function then they won't be able to call it like...

```kotlin
PinAction(true) {
    ...
}
```

As per your implementation, calling of method like the above isn't possible.

Also, in Kotlin and Compose, it's always good to have data parameters in the first place, and then have event (lambda) parameters later or at the end. I hope it makes sense.

Reply:
Oh Okay, Got it. Thanks.